pkgs/applications/misc/acpic/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, fetchPypi
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "acpic";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit version;
+    inherit pname;
+    sha256 = "sha256-vQ9VxCNbOmqHIY3e1wq1wNJl5ywfU2tm62gDg3vKvcg=";
```

```suggestion
    hash = "sha256-vQ9VxCNbOmqHIY3e1wq1wNJl5ywfU2tm62gDg3vKvcg=";
```

With an SRI hash you can and should prefer `hash`.

---

done